docs/sources/shared/find-and-use-plugins.md

```diff
@@ -0,0 +1,43 @@
+---
+aliases:
+  - /docs/grafana/fundamentals/find-and-use-plugins/
+  - /docs/grafana/how-do-i/find-and-use-plugins/
+  - /docs/grafana/introduction/find-and-use-plugins/
+description: Grafana Plugins
+labels:
+  products:
+    - oss
+keywords:
+  - Grafana OSS
+  - Plugins
+  - Install Plugins
+  - Download Plugins
+menuTitle: Find and use plugins
+title: Find and use plugins
+weight: 750
+---
+
+# Find and use Grafana plugins
+
+Plugins extend Grafana by adding capabilities that are not already present. There are plugins that add the ability to import data from diverse sources, that bundle data sources and panels, and that provide new visualization types for use in dashboards.
+
+To find new plugins, browse the Grafana plugins catalog which is available both [within Grafana](https://grafana.com/docs/grafana/latest/administration/plugin-management/#browse-plugins) and on the Grafana [website](/grafana/plugins/).
+
+### Install a plugin from the Grafana website
+
+1. Browse the [Grafana plugin catalog](/grafana/plugins) to find a plugin.
+1. Go to the **Installation** tab.
+1. Optional: select the desired version. We strongly recommended to run the latest version of plugins.
+1. Click **Install plugin** on the appropriate Grafana instance.
+
+### Install a plugin from Grafana
+
+1. In Grafana's right-hand menu, click **Administration** > **Plugins and data** > **Plugins** to view installed plugins.
+1. Click the **All** filter to browse all available plugins.
+1. Click the **Data sources**, **Panels**, or **Applications** buttons to filter by plugin type.
+1. Click the logo of the plugin you want to install.
+1. Click install.
```

Comment:
If this is a UI button, we should make sure that it's spelled the exact same way as in the UI and it should also be in bold https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold